packages/api-proxy/README.md

```diff
@@ -0,0 +1,156 @@
+# api-proxy
+
+Used for authentication and communication with a [contractsgenerator](https://github.com/leancodepl/contractsgenerator)
+based api.
+
+## Usage
+
+### register
+
+```ts
+import { VerifyOptions } from "jsonwebtoken";
+
+static register({ isGlobal = false, jwtStrategyConfig }: ApiProxySyncConfiguration): DynamicModule
```

Comment:
`isGlobal = false` is not exactly true anymore.
`static` in this context looks kinda weird. IMO it'd be better if you'd do `ApiProxy.register({ ...`